google-ads/build.gradle

```diff
@@ -62,14 +62,14 @@ protobuf {
 dependencies {
     api project(':google-ads-codegen')
     implementation 'org.slf4j:slf4j-api:1.7.25'
-    annotationProcessor 'com.google.auto.value:auto-value:1.10.1'
+    annotationProcessor 'com.google.auto.value:auto-value:1.10.2'
     testImplementation 'org.hamcrest:java-hamcrest:2.0.0.0'
     testImplementation 'org.mockito:mockito-all:1.9.5'
     testImplementation 'com.google.api:gax-grpc::testlib'
-    testImplementation 'io.grpc:grpc-context:1.55.1'
+    testImplementation 'io.grpc:grpc-context'
```

Comment:
What is the reason we don't want to pin this to a specific version?

Reply:
The only reason we were doing this was that the shared deps project wasn't properly declaring a version for `grpc-context` at one point, but when I inspect the dependencies report now, I see that's been fixed so I removed it. Now this works like the other dependencies in that it pulls the version that the shared deps artifact declares.